Makefile

```diff
@@ -491,42 +491,28 @@ test/karma-conf-dev.js: test/karma-conf.mako.js ${MAKO_CMD}
 test/karma-conf-prod.js: test/karma-conf.mako.js ${MAKO_CMD}
 	${PYTHON_CMD} ${MAKO_CMD} --var "mode=prod" $< > $@
 
-test/lib/angular-mocks.js:
-	cp -f node_modules/angular-mocks/angular-mocks.js $@;
-test/lib/expect.js:
-	cp -f node_modules/expect.js/index.js $@;
-test/lib/sinon.js:
-	cp -f node_modules/sinon/pkg/sinon.js $@;
-externs/angular.js:
-	cp -f node_modules/google-closure-compiler/contrib/externs/angular-1.4.js $@;
-externs/jquery.js:
-	cp -f node_modules/google-closure-compiler/contrib/externs/jquery-1.9.js $@;
-.PHONY: package_json_dev
-package_json_dev:
-	npm install --only=dev
-node_modules: package_json_dev externs/jquery.js externs/angular.js test/lib/sinon.js test/lib/expect.js test/lib/angular-mocks.js
-
-ANGULAR_JS = angular.js angular.min.js
-ANGULAR_TRANSLATE_JS = angular-translate.js angular-translate.min.js
-ANGULAR_TRANSLATE_LOADER_JS = angular-translate-loader-static-files.js angular-translate-loader-static-files.min.js
-LOCALFORAGE = localforage.js localforage.min.js
-JQUERY = jquery.js jquery.min.js
-JQUERYXDOMAIN = jQuery.XDomainRequest.js  jquery.xdomainrequest.min.js
-D3 = d3.js  d3.min.js
-BOOTSTRAP = bootstrap.js bootstrap.min.js
-SLIPJS = slip.js
-.PHONY: node_modules_prod
-node_modules_prod: node_modules
+test/lib/angular-mocks.js test/lib/expect.js test/lib/sinon.js externs/angular.js externs/jquery.js:
```

Comment:
If you add `package.json` as prerequisite here, the dev libraries will be updated when you change their version in the package.json.

And IMO, `libs` target should auto-update when changing package.json (would need specific target to have both forced/manual update and auto-update on package.json change)